pallets/ddc-clusters/src/benchmarking.rs

```diff
@@ -0,0 +1,113 @@
+//! DdcStaking pallet benchmarking.
+
+use ddc_primitives::{ClusterGovParams, ClusterId, ClusterParams, NodePubKey};
+pub use frame_benchmarking::{
+	account, benchmarks, impl_benchmark_test_suite, whitelist_account, whitelisted_caller,
+	BenchmarkError,
+};
+use frame_system::RawOrigin;
+use pallet_contracts::chain_extension::UncheckedFrom;
+use sp_runtime::{AccountId32, Perbill};
+use sp_std::prelude::*;
+use testing_utils::*;
+
+use super::*;
+use crate::{cluster::ClusterProps, Pallet as DdcClusters};
+
+const USER_SEED: u32 = 999666;
+const USER_SEED_2: u32 = 999555;
+
+benchmarks! {
+  where_clause { where
+		T::AccountId: UncheckedFrom<T::Hash> + AsRef<[u8]> }
+
+	create_cluster {
+	  let cluster_id = ClusterId::from([1; 20]);
+		let user = account::<T::AccountId>("user", USER_SEED, 0u32);
```

Comment:
Why do we have different indentations for these lines of code? Maybe the `fmt` formatting needs to be applied?